api-spec/definitions/STAC-standalone.yaml

```diff
@@ -1,355 +1,543 @@
+        Only features that have a geometry that intersects the bounding box are
+        selected. The bounding box is provided as four or six numbers,
+        depending on whether the coordinate reference system includes a
+        vertical axis (elevation or depth):
+
+        * Lower left corner, coordinate axis 1
+        * Lower left corner, coordinate axis 2  
+        * Lower left corner, coordinate axis 3 (optional) 
+        * Upper right corner, coordinate axis 1 
+        * Upper right corner, coordinate axis 2 
+        * Upper right corner, coordinate axis 3 (optional)
+
+        The coordinate reference system of the values is WGS84
+        longitude/latitude (http://www.opengis.net/def/crs/OGC/1.3/CRS84) unless
+        a different coordinate reference system is specified in the parameter
+        `bbox-crs`.
+
+        For WGS84 longitude/latitude the values are in most cases the sequence
+        of minimum longitude, minimum latitude, maximum longitude and maximum
+        latitude. However, in cases where the box spans the antimeridian the
+        first value (west-most box edge) is larger than the third value
+        (east-most box edge).
+
+
+        If a feature has multiple spatial geometry properties, it is the
+        decision of the server whether only a single spatial geometry property
+        is used to determine the extent or all relevant geometries.
+      required: false
+      schema:
+        type: array
+        minItems: 4
+        maxItems: 6
+        items:
+          type: number
+      style: form
+      explode: false
+    time:
+      name: time
+      in: query
+      description: >
+        Either a date-time or a period string that adheres to RFC3339. Examples:
+
+        * A date-time: "2018-02-12T23:20:50Z" 
+
+        * A period: "2018-02-12T00:00:00Z/2018-03-18T12:31:12Z" or
+
+        "2018-02-12T00:00:00Z/P1M6DT12H31M12S" 
+
+
+        Only features that have a temporal property that intersects the value
+        of 
+
+        `time` are selected. If a feature has multiple temporal properties, it
+        is
+
+        the decision of the server whether only a single temporal property is
+        used to 
+
+        determine the extent or all relevant temporal properties.
+      required: false
+      schema:
+        type: string
+      style: form
+      explode: false
+    collectionId:
+      name: collectionId
+      in: path
+      required: true
+      description: Identifier (name) of a specific collection
+      schema:
+        type: string
+    featureId:
+      name: featureId
+      in: path
+      description: Local identifier of a specific feature
+      required: true
+      schema:
+        type: string
+  schemas:
+    exception:
+      type: object
+      required:
+        - code
+      properties:
+        code:
+          type: string
+        description:
+          type: string
+    link:
+      type: object
+      required:
+        - href
+      properties:
```

Comment:
As below, I'd add `additionalProperties: true` to allow other fields such as type and title in the link, too.